src/ui/common/page/PageTree.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import TreeNodeFolderIcon from 'ui/common/tree-node/TreeNodeFolderIcon';
+import TreeNodeExpandedIcon from 'ui/common/tree-node/TreeNodeExpandedIcon';
+import RowSpinner from 'ui/common/RowSpinner';
+
+
+class PageTree extends Component {
```

Comment:
please rename it in order to avoid confusion between the appbuilder PageTree and this one